clients/python/generate_schema_types.py

```diff
@@ -0,0 +1,498 @@
+#!/usr/bin/env python3
+"""
+Generate Python dataclasses from Rust JSON schemas.
+
+This script:
+1. Runs cargo tests to generate JSON schemas from Rust types (using #[export_schema])
+2. Generates Python dataclasses from those JSON schemas
```

Comment:
can you elaborate on how this works here?

Reply:
Done.